packages/sanity/src/core/form/inputs/PortableText/Editor.tsx

```diff
@@ -31,6 +31,8 @@ import {useScrollSelectionIntoView} from './hooks/useScrollSelectionIntoView'
 import {Style} from './text/Style'
 import {ListItem} from './text/ListItem'
 
+const noOutlineStyle: React.CSSProperties = {outline: 'none'}
```

Comment:
Consider removing the dependency to React.CSSProperties by changing to

```suggestion
const noOutlineStyle = {outline: 'none'} as const
```